src/components/Layout/Layout.tsx

```diff
@@ -0,0 +1,17 @@
+import { Flex } from "@adobe/react-spectrum";
+import React, { FC } from "react";
+
+import { Header } from "../Header";
+
+interface Props {}
+
+const Layout: FC<Props> = ({ children }) => {
```

Comment:
Como esté componente no require ningun prop especial, podemos quitar este empty interface 

(`children` es un prop que los **F**unction **C**omponents (FC) tienen, entonces no hay que definirlo en el interface)
```suggestion


const Layout: FC = ({ children }) => {
```